mysql/src/lib.rs

```diff
@@ -561,18 +561,21 @@ impl<B: AsyncMysqlShim<Cursor<Vec<u8>>> + Send + Sync, S: AsyncRead + AsyncWrite
                     // NOTE: spec dictates no response from server
                 }
                 Command::ListFields(_) => {
-                    let cols = &[Column {
-                        table: String::new(),
-                        column: "not implemented".to_owned(),
-                        coltype: myc::constants::ColumnType::MYSQL_TYPE_SHORT,
-                        colflags: myc::constants::ColumnFlags::UNSIGNED_FLAG,
-                    }];
-                    writers::write_column_definitions_41(
-                        cols,
-                        &mut self.writer,
-                        self.client_capabilities,
-                        true,
-                    )?;
+                    // FIXME: unimplemented
+                    // BUG: Invalid column definitions lead to incorrect mariadb-client behaviour
+                    // https://github.com/datafuselabs/databend/issues/4439
+                    // let cols = &[Column {
```

Comment:
I prefer to remove not used code instead of commenting it out.

How about adding detailed TODO notes here instead?